### ISSUE TITLE
add some hints that a node as animation

### ISSUE THREAD

**Issue by sozap** [MrKepzie/Natron#1400]
### Problem

**Expected behavior:** 
When a node as animation that would be cool to see something in the graph. In nuke it's a little A in the corner of the node...
http://3.bp.blogspot.com/-9slOI9Tzg70/UlkXDygiQwI/AAAAAAAADBw/01szzd-SVv8/s1600/trackRoto_0857.jpg

also there is the same issue with nodes and expressions : 
https://www.thefoundry.co.uk/products/nuke/developers/63/pythondevguide/_images/animPath_01.png

**Actual behavior:**
For now there is no information that a node as some values animated, on a complex nodegraph I must change node colors or name them to help me see which node is animated .

There is a little P that appear when a merge node as only the B input connected, but I think it's more important to see at least animation...

### Versions
Natron version 2.1.2 Release.
OS : linux 64bits

Cheers !

**Comment by sozap** [comment]
expressions indeed show a little green E hint, but only when copying - linked a value. 
Standard python expression don"t trigger the green E...

**Comment by MrKepzie** [comment]
They do, except if this is an expression on the same node or linking to the Group node if the expression is on a node wishin the sub-graph. Can you show an example where this doesn't work ?

**Comment by sozap** [comment]
Hello,

This is what I get with linked values, animations, and expressions, only linked values show something...
That would be very cool to have something that show animation...

![hints](https://cloud.githubusercontent.com/assets/9614046/17249673/55dd9986-55a1-11e6-839e-018b1e1066ed.png)

**Comment by sozap** [comment]
Sorry, it seem I've misunderstand you when you said : 
> except if this is an expression on the same node

In fact you were right, 

That said, I found that behaviour a bit confusing, let's say for instance that I create an expression based on current frame value rather than using another nodes parameters, then I've got an expression on that node, but no information about it. 
This could be helpful to see the green E even in that case.

While I agree that may not be top priority, these kind of detail help users a lot to feel more comfortable , that's especially true when working on big nodegraph and or on the project of someone else.

I'll make a pray to the CG gods so they can send you one or two interns to work on these kind of stuff :D
